test/test_b2http.py

```diff
@@ -195,3 +196,40 @@ def test_get_content(self):
             self.assertTrue(self.response is r)  # no assertIs until 2.7
         self.session.get.assert_called_with(self.URL, headers=self.EXPECTED_HEADERS, stream=True)
         self.response.close.assert_called_with()
+
+
+class TestClockSkewHook(TestBase):
+    def test_bad_format(self):
+        response = MagicMock()
+        response.headers = {'Date': 'bad format'}
+        with self.assertRaises(BadDateFormat):
+            ClockSkewHook().post_request('POST', 'http://foo.com', {}, response)
```

Comment:
there are several `foo.com` instances left in this class, please change it to `example.com`